### PR TITLE
Endpoint for script_execution

### PR DESCRIPTION
**Pull request by Levizar**
**Is your feature request related to a problem? Please describe.**
Create an endpoint API to retrieve the script_execution by runId and ProcessId

**Id** :
a833065

### CODE REVIEW

File: core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/scriptExecutionDto/tools/ActionExecutionDtoBuildHelper.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Map;
+
+@Builder
+@Data
+@AllArgsConstructor
+public class ActionExecutionDtoBuildHelper {
```
Comment (BerrevoetsRobbe) [inline core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/scriptExecutionDto/tools/ActionExecutionDtoBuildHelper.java:15]: Please create inner class of this in ScriptDtoRepository

File: core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/scriptExecutionDto/tools/ScriptExecutionDtoBuildHelper.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Builder
+@Data
+@EqualsAndHashCode(callSuper = false)
+@AllArgsConstructor
+@NoArgsConstructor
+public class ScriptExecutionDtoBuildHelper {
```
Comment (BerrevoetsRobbe) [inline core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/scriptExecutionDto/tools/ScriptExecutionDtoBuildHelper.java:20]: Please create inner class of this in ScriptDtoRepository

File: core/java/iesi-rest-without-microservices/src/test/java/io/metadew/iesi/server/rest/scriptExecutionDto/ScriptExecutionDtoServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = Application.class, properties = {"spring.main.allow-bean-definition-overriding=true"})
+@ContextConfiguration(classes = TestConfiguration.class)
+@ActiveProfiles("test")
+class ScriptExecutionDtoServiceTest {
```
Comment (BerrevoetsRobbe) [inline core/java/iesi-rest-without-microservices/src/test/java/io/metadew/iesi/server/rest/scriptExecutionDto/ScriptExecutionDtoServiceTest.java:48]: comment out for now

File: core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/scriptExecutionDto/ScriptExecutionDto.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    private Long parentProcessId;
+    private String scriptId;
+    private String scriptName;
+    private Long scriptVersion;
+    private String environment;
+    private ScriptRunStatus status;
+    private LocalDateTime startTimestamp;
+    private LocalDateTime endTimestamp;
+    private List<ExecInputParameterDto> inputParameters = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (Levizar) [reply core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/scriptExecutionDto/ScriptExecutionDto.java:34]: ExecInputParameterDto

**Pull request merged**